.github/workflows/build-test.yaml

```diff
@@ -26,6 +26,7 @@ on:
     - 'testdata/**'
   push:
     branches: main
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
This was probably meant to be under the pull request section above
```suggestion
```

Reply:
Yes, indeed. Fixed now